### PR TITLE
Allow multi DB tests to be disabled by environment variable

### PR DESCRIPTION
**Pull request by darrellwarde**
# Description

> **Note**
> 
> Please provide a description of the work completed in this PR below

Allows us to disable multi DB for integration tests when running against Aura.

## Complexity

> **Note**
>
> Please provide an estimated complexity of this PR of either Low, Medium or High

Complexity: Low

### CODE REVIEW

**Comment by neo4j-team-graphql** [comment]
# Performance Report

No Performance Changes


<details>
    <summary>Show Full Table</summary>
    
| name | dbHits | old dbHits | time (ms) | old time (ms) | maxRows |
| ---- | ------ | ---------- | --------- | ------------- | ------- |
|  aggregations.TopLevelAggregate | 3403 | 3403 | 35 | 42 | 1134 |
|  aggregations.NestedAggregation | 13514 | 13514 | 97 | 112 | 2174 |
|  aggregations.AggregationWithWhere | 10942 | 10942 | 84 | 68 | 2174 |
|  aggregations.AggregationWhereWithinNestedRelationships | 20101954 | 20101954 | 2541 | 2809 | 2008534 |
|  aggregations.AggregationWhereWithinNestedConnections | 20101954 | 20101954 | 2501 | 2465 | 2008534 |
|  aggregations.NestedCountFromMovieToActors | 8694 | 8694 | 62 | 114 | 2174 |
|  aggregations.NestedCountFromActorsToMovie | 8900 | 8900 | 56 | 105 | 2174 |
|  aggregations.DeeplyNestedCount | 10054408 | 10054408 | 4217 | 3894 | 2008534 |
|  batch-create.BatchCreate | 3600 | 3600 | 97 | 120 | 600 |
|  connect.createAndConnect | 12419 | 12419 | 330 | 321 | 3003 |
|  connections.Connection | 13042 | 13042 | 139 | 109 | 2174 |
|  connections.NestedConnection | 33883 | 33883 | 112 | 140 | 4516 |
|  connections.ConnectionWithSort | 2277 | 2277 | 56 | 66 | 1040 |
|  connections.ConnectionWithSortAndCypher | 2277 | 2277 | 161 | 236 | 1040 |
|  create.SimpleMutation | 6 | 6 | 30 | 54 | 1 |
|  cypher-directive.TopLevelCypherDirective | 99525 | 99525 | 679 | 759 | 10168 |
|  cypher-directive.TopLevelCypherDirectiveWithColumnName | 127992 | 127994 | 290 | 313 | 10168 |
|  delete.SimpleDelete | 18361 | 18361 | 336 | 272 | 1040 |
|  delete.NestedDeleteInUpdate | 16779 | 16779 | 126 | 222 | 2040 |
|  query.SimpleQuery | 14120 | 14120 | 84 | 129 | 2174 |
|  query.QueryWhere | 8657 | 8659 | 32 | 39 | 2159 |
|  query.SimpleQueryWithNestedWhere | 8828 | 8830 | 46 | 53 | 2159 |
|  query.Nested | 10084290 | 10084290 | 10185 | 10235 | 2008534 |
|  query.NestedWithFilter | 37172 | 37172 | 133 | 157 | 4511 |
|  query.OrFilterOnRelationships | 36246 | 36932 | 153 | 196 | 1699 |
|  query.OrFilterOnRelationshipsAndNested | 30308 | 31002 | 186 | 271 | 1699 |
|  query.QueryWithNestedIn | 12237 | 12953 | 59 | 70 | 1179 |
|  query.NestedConnectionWhere | 8794 | 8794 | 49 | 67 | 2174 |
|  query.DeeplyNestedConnectionWhere | 8998 | 8898 | 73 | 82 | 2174 |
|  query.DeeplyNestedWithRelationshipFilters | 6251 | 6251 | 112 | 140 | 1134 |
|  query.NestedWithRelationshipSingleFilters | 3021189 | 3021189 | 692 | 655 | 1003942 |
|  query.Fulltext | 96 | 96 | 24 | 29 | 16 |
|  query.FulltextWithNestedQuery | 571 | 571 | 39 | 69 | 84 |
|  sorting.SortMultipleTypes | 2493 | 2493 | 59 | 182 | 1040 |
|  sorting.SortMultipleTypesWithCypherWithCypher | 1451 | 1457 | 149 | 217 | 1040 |
|  sorting.SortOnNestedFields | 13042 | 13042 | 63 | 107 | 2174 |
|  sorting.SortDeeplyNestedFields | 38965 | 38965 | 106 | 133 | 4516 |
|  sorting.SortWithTopLevelCypher | 2119 | 2119 | 91 | 145 | 1040 |
|  unions.SimpleUnionQuery | 321 | 321 | 53 | 59 | 35 |
|  unions.SimpleUnionQueryWithMissingFields | 293 | 293 | 51 | 60 | 35 |
|  unions.NestedUnion | 398553 | 398553 | 319 | 343 | 33033 |
|  unions.NestedUnionWithMissingFields | 372527 | 372527 | 272 | 336 | 33033 |
|  update.NestedUpdate | 2119 | 2119 | 42 | 56 | 1040 |

</details>
        
Old Schema Generation: 28.407s
Schema Generation: 28.310s

File: .github/workflows/reusable-aura-tests.yml [hunk 1]
```diff
@@ -75,6 +75,7 @@ jobs:
           yarn test:int
         working-directory: packages/${{ matrix.package }}
         env:
+          DEFAULT_DB: true
```
Comment (Liam-Doodson) [inline .github/workflows/reusable-aura-tests.yml:78]: Is DEFAULT_DB the best name for this? To me it's not clear it has anything to do with multi db

**Review by Liam-Doodson — APPROVED** [review]
LGTM except I'm unsure about the var name

File: .github/workflows/reusable-aura-tests.yml (hunk 1 above)
Comment (darrellwarde) [reply .github/workflows/reusable-aura-tests.yml:78]: Well it's not anything to do with multi DB really, it's just telling the tests to use the default DB for the DBMS - I figured this was better than `DISABLE_MULTI_DB` or something.

Comment (Liam-Doodson) [reply .github/workflows/reusable-aura-tests.yml:78]: Could potentially be `USE_DEFAULT_DB` then? Although it's not super important

Comment (darrellwarde) [reply .github/workflows/reusable-aura-tests.yml]: Sure, I've done that!!

**Pull request merged**